setup.py

```diff
@@ -203,6 +203,7 @@
             'inspire_cache = inspirehep.modules.cache.ext:INSPIRECache',
             'inspire_search = inspirehep.modules.search:INSPIRESearch',
             'inspire_workflows = inspirehep.modules.workflows:INSPIREWorkflows',
+            'invenio_collections = invenio_collections:InvenioCollections',
```

Comment:
> You are testing the PUT request here: https://github.com/inspirehep/inspire-next/pull/1678/files#diff-8eedb14f28e016fe0e021789e25878abR348

Indeed, removing `invenio-collections` should make that test fail. I don't know if it is useful to do though. The reason why it is needed is because the PUT request will try to update the record in DB, the signal `before_record_update` of `invenio-collections` will kick in but the API app does not have the `invenio-collections` extension so it fails.

But ok, @zzacharo you can comment it out and `push -f` to see it failing on Travis.